cli/cook/querying.py

```diff
@@ -330,7 +334,8 @@ def parse_entity_refs(clusters, ref_strings):
                 for uuid in query_args['uuid']:
                     entity_refs.append({'cluster': cluster_name, 'type': entity_type, 'uuid': uuid})
 
-    return entity_refs
+    clusters_of_interest = [c for c in clusters if c['name'] in cluster_names_of_interest]
+    return entity_refs, clusters_of_interest
```

Comment:
You should probably update the docstring to reflect the new (pair) return type.

Reply:
Good call, done